contracts/Oracle.sol

```diff
@@ -24,9 +24,13 @@ contract Oracle is Ownable {
 
     /// @dev mapping of asset pricer to its locking period
     /// locking period is the period of time after the expiry timestamp where a price can not be pushed
+    /// the locking period is set per Pricer, it can be set to zero in case the system have a pricer with an instant asset price
```

Comment:
This explanation doesn't make much sense to me, it's unclear what **instant asset price** mean. 🤔 
`in case the system have a pricer with an instant asset price`